packages/core/src/Stores/client-store.js

```diff
@@ -215,6 +215,36 @@ export default class ClientStore extends BaseStore {
         return false;
     }
 
+    @computed
+    get is_spending_limit_visible() {
+        if (!this.loginid || !this.landing_company) {
+            return false;
+        }
+
+        const balance = parseFloat(this.balance);
+
+        if (!this.has_reality_check) return false;
+
+        if (
+            this.has_reality_check &&
```

Comment:
This is unnecessary since we already know from `line 226` that `has_reality_check` is true!

Reply:
@matin-binary just fixed.